packages/ui-kit/src/components/multi-code-block.tsx

```diff
@@ -193,23 +193,25 @@ function MultiCodeBlock(props: MultiCodeBlockProps) {
         })}
     </Container>
   );
-}
-
-export default MultiCodeBlock;
+};
 
 const getCodeBlockPropsFromMdxPreNodeProps = (props: {
   children?: React.ReactNode;
 }) => {
-  const childElement = reactIs.isElement(props.children)
-    ? props.children
-    : null;
+  const childElement =
+    props.children && reactIs.isElement(props.children) ? props.children : null;
```

Comment:
This can also be simplified:
```
const childElement = reactIs.isElement(props?.children) ? props.children : null;
```
As the `reactIs.isElement(undefined)` I assume will return false